src/Build/GlobalAssemblyInfo.cs

```diff
@@ -1,3 +1,29 @@
+// Project Orleans Cloud Service SDK ver. 1.0
```

Comment:
Formatting tools apply code style formatting in some "standard" way, partly to allow easy updates later.

FWIW, the Rosylyn CodeFormatter does that the same way as ReSharper is doing here - using line comments rather than block comments.
https://github.com/dotnet/codeformatter/blob/master/src/Microsoft.DotNet.CodeFormatting/Rules/CopyrightHeaderRule.cs#L68